pydss/flapper.py

```diff
@@ -108,10 +108,10 @@ def file(self,what, data):
             self.tau = data;
         else:
             raise Exception("Flapper/file-unrecognized-param");
-    def deal(self,bid_id):
+    def deal(self,bid_id,sender,bid):
         require(self.live == 1,"Flapper/not-live")
         require(self.bids[bid_id].tic != 0 and (self.bids[bid_id].tic < self.now or self.bids[bid_id].end < self.now),"Flapper/not-finished")
-        self.vat.move(bid_id, self.bids[bid_id].guy, self.bids[bid_id].lot);
+        self.gem.transferFrom(sender, self.ADDRESS, bid - self.bids[bid_id].bid)
```

Comment:
ya u can pretty much just undo this change and the change to the function parameters as per the comment above re: `vat.move`